sssom/util.py

```diff
@@ -1040,6 +1036,18 @@ class NoCURIEException(ValueError):
 
 
 CURIE_RE = re.compile(r"[A-Za-z0-9_.]+[:][A-Za-z0-9_]")
+schema_view_object = SSSOMSchemaView()
```

Comment:
this is again in the global context I think.. wont this be invoked any time anyone imports anything from utils package?

Reply:
I will have to repeat this code in 4 different lines in the same file. Seems messy.